packages/components/src/keyboard-shortcuts/index.js

```diff
@@ -21,7 +35,12 @@ class KeyboardShortcuts extends Component {
 		const { keyTarget = document } = this;
 
 		this.mousetrap = new Mousetrap( keyTarget );
+
 		forEach( this.props.shortcuts, ( callback, key ) => {
+			if ( process.env.NODE_ENV === 'development' ) {
+				throwForInvalidCombination( key );
```

Comment:
FWIW, if the logic of the function was inlined here, this could benefit from dead code removal for production environments.

https://gist.github.com/aduth/f6e7ae516310f60ebda6c93998713254

Reply:
done it.